### PR TITLE
FIX: translate bootbox confirmation dialog

### PR DESCRIPTION
**Pull request by featheredtoast**
Localize bootbox confirmation buttons - pull from Discourse translation keys

### CODE REVIEW

File: app/assets/javascripts/discourse/initializers/localization.js.es6 [hunk 1]
```diff
@@ -51,5 +51,12 @@ export default {
         node[segs[segs.length - 1]] = v;
       }
     });
+
+    bootbox.addLocale(I18n.currentLocale(), {
+      OK: I18n.t("composer.modal_ok"),
+      CANCEL: I18n.t("composer.modal_cancel"),
+      CONFIRM: I18n.t("composer.modal_cancel")
```
Comment (ZogStriP) [inline app/assets/javascripts/discourse/initializers/localization.js.es6]: Confirm == cancel?

**Comment by eviltrout** [comment]
Don't we normally pass these values to bootbox? Is this for the case when defaults are not supplied?

I suppose if defaults are supplied we could go back and refactor things to remove extra code.

File: app/assets/javascripts/discourse/initializers/localization.js.es6 (hunk 1 above)
Comment (featheredtoast) [reply app/assets/javascripts/discourse/initializers/localization.js.es6]: oops, good catch! was supposed to be modal_ok 🤦‍♂️

**Comment by featheredtoast** [comment]
yeah, I think this is for cases where the buttons are not supplied, which we do at least in ajax-error.js

**Comment by SamSaffron** [comment]
Cool, yeah this is a good change. That said I think we should look at removing bootbox and putting our own abstraction in place longer term that is built in ember and so on so we can control our destiny.

**Pull request merged**